deploy-renku/entrypoint.sh

```diff
@@ -12,8 +12,14 @@ echo $DOCKER_PASSWORD | docker login -u $DOCKER_USERNAME --password-stdin
 # set up kube context and values file
 echo "$RENKUBOT_KUBECONFIG" > "$KUBECONFIG" && chmod 400 "$KUBECONFIG"
 
-# set up the values file
-printf "%s" "$RENKU_VALUES" | sed "s/<replace>/${RENKU_RELEASE}/" > $RENKU_VALUES_FILE
+export RENKU_VALUES_FILE="/renku-values.yaml"
+
+# merge the secret and clear value files
+printf "%s" "$RENKU_SECRET_VALUES" > $RENKU_VALUES_FILE
```

Comment:
We need to update the README.md for these changes accordingly. According to the README the secret values are called just `RENKU_VALUES`.